src/utils/constants.js

```diff
@@ -39,3 +39,4 @@ export const BUGZILLA_LANGUAGES = {
   XML: 'xml',
   XUL: 'xul',
 };
+export const GITHUB_LANGUAGE_QUERY_ORDER = { field: 'SIZE', direction: 'DESC' };
```

Comment:
What does `field: 'SIZE'` mean? Can you show me a link online to where this is explained, please?

Reply:
https://developer.github.com/v4/object/repository/ go to the languages section, it basically says that it will return the languages a repository has based on its size (starting from the major ones to the minor ones like html). and we use this later when we add field `first: 1` to be only showing the first (only show the most used language)

Reply:
Makes sense. Thanks for the explaining.

Reply:
@fiennyangeln There seems to be a `primaryLanguage` field. Shouldn't we use this instead of doing `first: 1`? https://developer.github.com/v4/object/repository/#fields